packager/media/formats/mp4/track_run_iterator.cc

```diff
@@ -451,6 +455,17 @@ void TrackRunIterator::ResetRun() {
   sample_dts_ = run_itr_->start_dts;
   sample_offset_ = run_itr_->sample_start_offset;
   sample_itr_ = run_itr_->samples.begin();
+
+  int64_t min_cts_offset(0);
+  auto min_runs =
```

Comment:
nit: This iterator points to samples, so something like `min_samp` or `samp_itr` might be a better name.